### DISCUSSION TITLE
Calling Component with Multiple Markdown Props Elements

### DISCUSSION THREAD

**Discussion by CesarJSmokowski** [mdx-js/mdx#1958] [Q&A] [answered]
I am working on a research project where we just integrated MDX v2 into our website directory. I'm trying to create an education page from my education.mdx file, where I'm calling my CustomCard component with two markdown elements, one for the name and another for the description. The instance where I'm calling the CustomCard component is shown below

`<CustomCard name="Supercomputers" label={<>*Super Computers*</>} 
          description={<>## A supercomputer is a network of computers that together have a superior level of performance. Though this performance can be...</>} 
          href="/education/supercomputer" image="/static/images/education/supercomputer.png"> *Content* </CustomCard>`

My CustomCard component recognizes props.children as a markdown element, but not the the props.label or props.description values. Is there a better way to pass in two markdown elements to a component? Thank you for the help!

**Comment by ChristianMurphy** [comment, marked as answer]
It sounds like you have similar interests and needs to https://github.com/mdx-js/mdx/discussions/1954
The options in https://github.com/mdx-js/mdx/discussions/1954#discussioncomment-2262015 offer a few options which may solve your use case as well
